script.js

```diff
@@ -1 +1,146 @@
-console.log('Hello world');
+const ham = document.getElementById('ham');
+const mobile = document.querySelector('.mobiles-menu');
+const hide = document.querySelector('#hide');
+const links = document.querySelectorAll('li');
+
+mobile.classList.add('removed');
+hide.style.display = 'none';
+
+ham.addEventListener('click', () => {
+  mobile.style.display = 'block';
+  hide.style.display = 'block';
+  ham.style.display = 'none';
+  ham.classList.add('remove');
+});
+
+hide.addEventListener('click', () => {
+  mobile.style.display = 'none';
+  hide.style.display = 'none';
+  ham.style.display = 'block';
+});
+
+links.forEach((link) => {
+  link.addEventListener('click', () => {
+    mobile.style.display = 'none';
+    hide.style.display = 'none';
+    ham.style.display = 'block';
+  });
+});
+
+const speakersData = [
+  {
+    id: '01',
+    name: 'Yochai Benkler',
+    role: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    description:
+        'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006 ',
+    featuredImage: 'Img/speaker_01 1 (1).svg',
+  },
+
+  {
+    id: '02',
+    name: 'SohYeong Noh',
+    role: 'Director of Art Centre Nabi and a board member of CC Korea',
+    description:
+        'As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.',
+    featuredImage: 'Img/speaker_01 1 (2).svg',
+  },
+];
```

Comment:
- [ ] Please ensure that you have one array of objects to store all the speakers information for this particular section. This makes the code more easy to understand and also maintain👍. Therefore I suggest, you consider merging all the array of objects of the speakers into one ☺️.

Reply:
But is this main requirement. I think it should have been optional as there is nothing mentioned like that on the projects requirements

Reply:
- [ ] @Zel-hub7 Yes the requirement asked is part of the project requirements as shown below, we are required to always follow javascript best practice, creating different arrays for the same information is repetition, we should only have a single array to handle similar data, this will help in future scaling of the website and also fixing of the website error. 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/61408860/4b5ef6e9-ebc4-489e-9b6b-29e5845fe417)